.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 logs
+.DS_Store
```

Comment:
Out of curiosity: what is this?

Reply:
Haha, it's a Mac-specific file that stores custom configurations for directories.

Reply:
It's hidden and exists on all Macs.

Reply:
Please consider to put such stuff into your local `~/.config/git/gitignore` file. But since we have multiple Mac users lets keep it here.

Reply:
Ah, ok 👍